### ISSUE TITLE
LFO Selection when Adding via Menu

### ISSUE THREAD

**Issue by baconpaul** [surge-synthesizer/surge#6701]
Chessel — Today at 5:10 PM

Hi all.  I have a suggestion to aid usability for keyboard users - the typing keyboard that is.  If I set up a parameter with modulation, could the LFO controls region switch to showing that modulator?  So if I add a modulator to my osc 1 pitch using voice 1, if I then alt full stop to the LFO controls region, it should be on the voice 1 controls.  If I then added a filter, and modulation of the cut off frequency with voice 2, and I go to the LFO controls, it would have shifted to the voice 2 controls.  So whenever I set up a new modulation the focus of the LFO controls region shifts to show that modulator.  This would save having to go to the modulator region to select the LFO.

baconpaul — Today at 5:15 PM

oh so like "add modulation from lfo1" in the menu selects lfo1?
that's a bit of a pain for the signted ui since that menu is meant to allow you to add modulators from "away" sources while looking at "current" sources. but i see why you want it.

Chessel — Today at 5:16 PM

Yes. Exactly.

baconpaul — Today at 5:17 PM

OK let me drop it in a github issue

**Comment by mkruselj** [comment]
Yeah this should be an option off by default IMO.

**Comment by baconpaul** [comment]
Yeah that’s how the convo continued
